PaymentButtons/src/main/res/layout/paypal_ui_payment_button_view.xml

```diff
@@ -14,7 +14,7 @@
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
         android:layout_marginEnd="@dimen/paypal_payment_button_text_margin"
-        android:fontFamily="@font/paypal_sans_small_medium"
```

Comment:
qq: Are we able to remove [paypal_sans_small_medium.otf](https://github.com/paypal/paypal-android/blob/6e38b0b526d15baf24fbd21359673143d028c2d1/PaymentButtons/src/main/res/font/paypal_sans_small_medium.otf) with this change?

Reply:
I checked, apparently, we can, done:  https://github.com/paypal/paypal-android/pull/232/commits/39ca6cf15e94530b8e34af2e04a1add6c4d1dec8